src/shallow_copy.rs

```diff
@@ -78,6 +81,20 @@ impl<T> ShallowCopy for Vec<T> {
     }
 }
 
+#[cfg(feature = "bytes")]
+impl ShallowCopy for bytes::Bytes {
+    unsafe fn shallow_copy(&mut self) -> Self {
+        self.clone()
+    }
+}
+
+// This conflicts with the impl below for &'a T
+// impl<'a, T> ShallowCopy for &'a [T] {
```

Comment:
Is there a particular reason you want this implementation given that the one below already exists?